vulns/ultralytics/PYSEC-0000-ultralytics.yaml

```diff
@@ -0,0 +1,50 @@
+# TODO: Fill in the details below, and remove any comments prior to committing.
+# See https://ossf.github.io/osv-schema/ for format and field descriptions.
+id: PYSEC-0000-ultralytics.yaml
+modified: 2024-12-10T16:51:23Z
+summary: A number of releases of ultralytics contained malicious crypto miner software.
+details: |
+  Ultralytics has identified a supply chain attack
+  affecting affecting multiple versions of the ultralytics package.
+  The compromised versions contained unauthorized code that
+  downloaded and executed cryptocurrency mining software
+  when instantiating YOLO models.
+  This code was injected into the PyPI release artifacts and was not present
+  in the public GitHub repository.
+affected:
+- package:
+    ecosystem: PyPI
+    name: ultralytics
+    purl: pkg:pypi/ultralytics
+  versions:
+  - "8.3.41"
+  - "8.3.42"
+  - "8.3.45"
+  - "8.3.46"
+  ranges:
+  - type: ECOSYSTEM
+    events:
+      - introduced: "8.3.41"
```

Comment:
From what I gather from the spec, having a non-SEMVER range (aka ECOSYSTEM) and coupled with `versions` is the correct thing - are you reading it differently?
https://ossf.github.io/osv-schema/#affectedrangestype-field

Reply:
That's consistent with my understanding as well:

> It is recommended that you provide an explicitly enumerated versions list when specifying one or more ECOSYSTEM ranges, because ECOSYSTEM range inclusion queries may not be able to be answered without reference to the package ecosystem’s own logic and therefore may not be able to be used by ecosystem-independent processors. The infrastructure and tooling provided by https://osv.dev also provides automation for auto-populating the versions list based on supported ECOSYSTEM ranges as part of the ingestion process.